.github/containers/Dockerfile

```diff
@@ -22,8 +22,7 @@ RUN ./gradlew war -Phostedtest=true && \
     cp $(find ./build/libs -name 'candlepin*.war' | head -n 1) /app/build
 
 WORKDIR /app/build
-RUN jar xf $(find . -name 'candlepin*.war' | head -n 1); \
-    sed -i 's/jss4.jar/jss.jar/g' ./META-INF/context.xml
+RUN jar xf $(find . -name 'candlepin*.war' | head -n 1);
```

Comment:
The only reason for extracting the war file was to update the context.xml file, but if we don't have that file any more you don't need to extract at all (same for the release container image too)

Reply:
It doesn't run otherwise.

Reply:
Yes, but now that we don't need to make jss adjustments we can start deploying war directly.